### PR TITLE
[Update] Complete Your User Profile - Content text updates from Figma, Checkbox Group Status/Focus Style Update

### LINKED ISSUE #177: [Update] Complete Your User Profile - Field Inputs - Focus Styling(Error), Checkbox Group Focus Dotted Line
Additional 'Complete Your User Profile' figma and DSR changes will go here.

Follow-up to [https://github.com/cfpb/sbl-frontend/pull/146](https://github.com/cfpb/sbl-frontend/issues/120)

## Needed Changes

- Style(fix): Error Message Change - 'You must enter your first name to complete your user profile and access the ~~system~~ platform.'
- Style(fix): Accessibility Change - Institution selection well - dotted line goes around the LEI, TIN and RSSD ID fields as well.
- Style(fix): Removed interfering styles

### LINKED ISSUE #190: [Task] Implement Checkbox styling (Error/Warning/Success)
Requires DS and DSR updates first:
https://github.com/cfpb/design-system-react/issues/285
https://github.com/cfpb/design-system/issues/1883

### PR DESCRIPTION
**Pull request by shindigira**
Closes #177 
Closes #190 

Follow-up to [https://github.com/cfpb/sbl-frontend/pull/146](https://github.com/cfpb/sbl-frontend/issues/120)

## Changes

- Style(fix): Error Message Change - 'You must enter your first name to complete your user profile and access the ~~system~~ platform.'
- Style(fix): Accessibility Change - Institution selection well - dotted line goes around the LEI, TIN and RSSD ID fields as well.
- Style(fix): Removed interfering styles
- Style(feat): Implements the new Checkbox (error/success/warning) and Checkbox Focus from DS/DSR
- Feat(fix): 'Clear Form' no longer clears the list of associated financial institutions

## Local Dev Testing
- `yarn up -i design-system-react` to update DSR package with Checkbox status styling

## Potential Follow-Up PRs
- Margin change between the icon and text in Alerts
- Vertically centering in Alerts

## Screenshots
Focus: Red dotted border
Screenshot 2024-01-17 at 12 12 32 PM
Focus: dotted line around entire label
Screenshot 2024-01-17 at 12 00 36 PM

### CODE REVIEW

**Comment by natalia-fitzgerald** [comment]
@shindigira 
When I hover over the links in the form-level alert I'm seeing the cursor instead of the hand. I think it's because it's missing an href. Can we fix? 
Screenshot 2024-01-18 at 9 15 29 PM

**Comment by natalia-fitzgerald** [comment]
@shindigira 
Can you double check the error content against the Figma file? It looks like there are a number of discrepancies - where the deployed page isn't matching the Figma file. I have not made recent changes to the content so I guess this was missed until now. 

### Inconsistency between Figma content and deployed page content
- Form-level error links
- Field-level error alert - associate financial institution

**Comment by billhimmelsbach** [comment]
@natalia-fitzgerald @shindigira 

Don't know if you saw it on the channel, but the latest of this branch is on AWS now as of 12:30 PT / 3:30 ET. Thanks! 👍

**Review by natalia-fitzgerald — CHANGES_REQUESTED** [review]
@shindigira - This is looking great. I have one small adjustment and then it should be good to go. 

- Reduce the space between the bottom of the text introduction and the notification to 45px?

**Review by billhimmelsbach — CHANGES_REQUESTED** [review]
I think there's a "Clear form" bug that we'll want to fix, where clicking it removes all financial institution choices.


https://github.com/cfpb/sbl-frontend/assets/19983248/ac5998af-93d7-459d-a985-b21699e8fbfb

File: src/pages/ProfileForm/Step1Form/Step1Form.tsx [hunk 1]
```diff
@@ -90,6 +91,7 @@ function Step1Form(): JSX.Element {
 
   useEffect(() => {
     const dataCheckedState = formatDataCheckedState(afData);
+    console.log('dataCheckedState:', dataCheckedState);
```
Comment (billhimmelsbach) [inline src/pages/ProfileForm/Step1Form/Step1Form.tsx:94]:
```suggestion

```

**Review by billhimmelsbach — CHANGES_REQUESTED** [review]

**Review by billhimmelsbach — APPROVED** [review]
Looks good! Natalia is going to take a peek at it, then it should be good to go. 👍

**Review by natalia-fitzgerald — APPROVED** [review]
Great work @shindigira!

**Pull request merged**